targets/targets.json

```diff
@@ -8145,7 +8145,7 @@
         "is_disk_virtual": true,
         "extra_labels": ["TOSHIBA"],
         "macros": ["__TMPM4G9__"],
-        "supported_toolchains": ["GCC_ARM", "ARM", "IAR"],
+        "supported_toolchains": ["GCC_ARM", "ARMC5", "IAR"],
```

Comment:
why ARMC6 is not supported? A new target should support ARMC6 by default.

Reply:
TMPM4G9 is supported by both ARMC5 and ARMC6, but if we are running the `mbed export` command it's throwing below error:
```
TMPM4G9 not supported by uvision5
``` 
Hence, we have updated the `supported_toolchains` as only ARMC5.

Reply:
that error - isn't it because of cmsis pack missing ? 

This should be ARMC6 - so lets find out  why is it failing to export